packages/teams-js/test/public/media.spec.ts

```diff
@@ -655,11 +505,9 @@ describe('media', () => {
             expect(mediaAttachments).toBeFalsy();
             expect(mediaError.errorCode).toBe(ErrorCode.SIZE_EXCEEDED);
```

Comment:
@erinha This might be an example of testing for an SDK error thrown from synchronous code. We were trying to find examples of this yesterday for the unit testing guidelines you were writing

Reply:
lol, glad i can help

Reply:
Can we change this to `expect(mediaError).toEqual({ ErrorCode.SIZE_EXCEEDED });` ?